src/getWeb3.js

```diff
@@ -29,6 +29,7 @@ let getWeb3 = () => {
               errorMsg = constants.WRONG_NETWORK_MSG
               console.log('This is an unknown network.', netId)
           }
+          document.title = `${netIdName} - POA governance dApp`
```

Comment:
Let's rename to 
`${netIdName} - POA Network Governance DApp`
as suggested here https://github.com/poanetwork/poa-dapps-voting/pull/31/files/ab1761adf93cfd81c999fb66f1ec9a9ef8567e83#r158846016

Reply:
ok